### ISSUE TITLE
Softlock in the Sanctuary. Too many NPCs

### ISSUE THREAD

**Issue by GoobyFlipFlop** [Dinopony/alundra-randomizer#61]
![RandoSoftlock](https://user-images.githubusercontent.com/130947672/232363034-708b39fd-3f0b-4f6f-b258-05a9aad84509.png)

There are two copies of Ronan, and one is blocking the staircase. I can't leave, I'm locked into the cutscene/quest with Septimus, and going to a different room inside the Sanctuary doesn't despawn them.

**Comment by Dinopony** [comment]
This is not a normal occurrence, since Sanctuary should be fullly inaccessible in randomizer.
I'd need to find when it can be entered (what were the previous actions / dungeons that were done prior to acessing sanctuary) to be able to solve it.

**Comment by buuf** [comment]
Maybe after doing Lars Crypt? I know Septimus is available to speak with in the village then can be seen at the sanctuary thereafter to trigger you gettin in by going behind the place?

**Comment by GoobyFlipFlop** [comment]
> This is not a normal occurrence, since Sanctuary should be fullly inaccessible in randomizer. I'd need to find when it can be entered (what were the previous actions / dungeons that were done prior to acessing sanctuary) to be able to solve it.

With this being a few months ago, I don't exactly remember the chain of events. I remember I was on a hunt for the Spring Bean and Ice Rod so I had done almost everything I could without those two items. I definitely did talk to Septimus when he was with Meia and that let me go in through the back.

**Comment by Dinopony** [comment]
Whoa, I definitely didn't expect you to remember it and it's definitely my fault for taking care of it so late.
Sorry for that, life got in the way and working on a game like Alundra is no easy task :)

For the moment, I'll let the problem as it is since finding how to reach that state would be like searching a needle in a haystack, but if you ever encounter it again, try reporting on this issue what you did before :
- dungeons cleared
- story-related events / cutscenes

Thanks again for your help 🙏

**Comment by GoobyFlipFlop** [comment]
Of course, I completely understand! Next time I boot up a seed, I'll check with Septimus after every event/dungeon and let you know when it's accessible.

Thank you for your work on the rando, it's been a lot of fun.

**Comment by GoobyFlipFlop** [comment]
> Whoa, I definitely didn't expect you to remember it and it's definitely my fault for taking care of it so late. Sorry for that, life got in the way and working on a game like Alundra is no easy task :)
> 
> For the moment, I'll let the problem as it is since finding how to reach that state would be like searching a needle in a haystack, but if you ever encounter it again, try reporting on this issue what you did before :
> 
> * dungeons cleared
> * story-related events / cutscenes
> 
> Thanks again for your help 🙏

Just played my first seed since we talked about this. I checked with Meia and looked for Septimus after every dungeon. I assume what triggered Septimus to be in front of the Sanctuary was beating Magyscar, but I took notes leading up to that. Notes below, I hope it helps you find it.

--------------

"Meia talks about going to the Sanctuary at start of seed, but could not find Septimus, back not accessible.

Silver armlet, Bombs, Fire rod at Lurvy's shop. Nothing at the shop in Inoa.

Did Tarn manor, found power glove and flail, then Cliffs of Madness. Sanctuary not accessible, Meia said same thing.

Completed Coal Mine first, still cannot access the sanctuary, Meia says the same thing, still cannot find Septimus.

Found bouqet on overworld north of Waterwheel house, then did Lars' Crypt. Meia says same thing, could not find Septimus, back not accessible.

Magyscar next, no major items found in the dungeon. Meia says the same thing, Septimus standing in front of Sanctuary, Sanctuary now accessible."

**Comment by GoobyFlipFlop** [comment]
Here's two screenshots of the inventory as well, so you'll know all the items I had leading up to beating Magyscar.

![Inventory Page 1](https://github.com/Dinopony/alundra-randomizer/assets/130947672/2bbf3bad-b407-4d4f-b1df-c60c6ba7f594)
![inventory page 2](https://github.com/Dinopony/alundra-randomizer/assets/130947672/4d8f05c8-fff1-424c-af5e-eb8bf4cdfa37)

**Comment by GoobyFlipFlop** [comment]
Ran another seed where Septimus did not appear until I beat Magyscar and talked to Meia in the village. This time the dungeon chain was Tarn > Coal Mine > Secret Shrine > Lars' Crypt > Coastal Cave > Fairy Pond > Magyscar. It appears that Septimus does not show up until after Magyscar and then speaking to Meia.

**Comment by Prinny-Wizz** [comment]
Hi 👋 
I had a similar situation come up - I found Septimus in front of Sanctuary after completing Magyscar, but, the second Ronan didn't block me from going down the stairs (letting me finish the dungeon).
I'll be continuing that seed (as much as I can) later to  see if this has unintended consequences. For now the NPCs are blocking the way to Lars' Crypt, but I already completed that dungeon.

As far as I remember my dungeon order was _(apologies if I get the names wrong)_
Tarn's Manor
Desert Shrine
Coal Mine
The beach cave 
(Fire Manor / Ice Manor)
The Volcano
(Gambling den)
Lars' Crypt
Magyscar

As for cutscenes: I had triggered the defences at Nirude's Lair; all associated with the above dungeons; surprised Lurvy by dropping down his chimney; I think that's it?

(As an aside, now I know this is unintended I'll avoid breaking into Sanctuary in future 😆 I do wonder what effect grabbing the Life Vessel inside will have on the seed, since I assume that was meant to be randomized to be somewhere else 🤔)

Edit - I meant to mention I don't *remember* talking to Meia, but I might've done so at the very start of the randomizer (it was my first run, so I thought possibly all the villagers gave item hints).

Second edit - I realised Nava's Hut was closed due to story events, which meant I would have to at least complete Meia's Dream to progress _(which presumably would put Nava back at the top of the stairs when his hut was open again, I ended up abandoning the seed)_

I did remember Yustel's house opened up partway through the randomizer, but didn't notice after which dungeon.
